e2e/pages/administration/confirm/confirm.po.js

```diff
@@ -77,19 +81,22 @@ class ConfirmPage {
     $('//button[@id="process-pending-listing-' + pendingListingId + '"]').scrollAndClick();
   }
 
+  rejectListingcheckbox (chplId) {
+    $('//input[@id="reject-pending-listing-' + chplId + '"]').scrollAndClick();
```

Comment:
Could we do the same simplification with the other ones too?
```suggestion
    $('#reject-pending-listing-' + chplId).scrollAndClick();
```